Library/ViewModels/ConfirmDetailsViewModel.swift

```diff
@@ -319,9 +333,11 @@ public class ConfirmDetailsViewModel: ConfirmDetailsViewModelType, ConfirmDetail
       refTag
     )
 
-    let createCheckoutEvents = pledgeDetailsData
+    let createCheckoutEvents = Signal.combineLatest(isLoggedIn, pledgeDetailsData)
```

Comment:
Should we call this on userSessionStarted as well? I think that would probably fix the main issue I have with this flow (having to hit "continue" twice)

Reply:
What's a good RxSwift way to say "do this when userSessionStarted or if continueCTATappedProperty has been tapped by a logged in user"? I don't think just adding self.userSessionStarted to this combineLatest would work.

Reply:
Although I wonder if that might double-fire, hm.